sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -331,23 +324,10 @@ AZ_NODISCARD az_result az_iot_hub_client_telemetry_get_publish_topic(
  */
 
 /**
- * @brief Gets the MQTT topic filter to subscribe to cloud to device requests.
+ * @brief The MQTT topic filter to subscribe to cloud to device requests.

```

Comment:
nit: Consider making this more readable. I am having difficulty parsing this :p

Maybe this?
```suggestion
 * @brief The MQTT topic filter to subscribe to 'cloud-to-device' requests.
```